packages/mdc-theme/_color-palette.scss

```diff
@@ -207,7 +207,7 @@ $material-color-yellow-700: #fbc02d;
 $material-color-yellow-800: #f9a825;
 $material-color-yellow-900: #f57f17;
 $material-color-yellow-a100: #ffff8d;
-$material-color-yellow-a200: #ffff00;
+$material-color-yellow-a200: #ff0;
```

Comment:
this changes seems to have sneaked into this PR...can you make sure packages/mdc-theme/_color-palette.scss is the exact same as our master branch?